benchmarks/test_benchmark_coo.py

```diff
@@ -11,6 +11,23 @@
 SEED = 42
```

Comment:
I'd import this from a central place, so it can be changed just once.

Reply:
Ok